src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -25,24 +25,24 @@ internal static class IDEDiagnosticIds
         public const string InlineAsTypeCheckId = "IDE0019";
         public const string InlineIsTypeCheckId = "IDE0020";
 
-        public const string UseExpressionBodyForConstructorsDiagnosticId = "IDE0020";
-        public const string UseExpressionBodyForMethodsDiagnosticId = "IDE0021";
-        public const string UseExpressionBodyForConversionOperatorsDiagnosticId = "IDE0022";
-        public const string UseExpressionBodyForOperatorsDiagnosticId = "IDE0023";
-        public const string UseExpressionBodyForPropertiesDiagnosticId = "IDE0024";
-        public const string UseExpressionBodyForIndexersDiagnosticId = "IDE0025";
-        public const string UseExpressionBodyForAccessorsDiagnosticId = "IDE0026";
+        public const string UseExpressionBodyForConstructorsDiagnosticId = "IDE0021";
```

Comment:
NOTE that we already have repos such as roslyn-project-system where we escalate all IDE suggestions to errors using the IDs. Tag @davkean as he did this work and will probably have to update our rulesets.